scripts/gen_help_html.lua

```diff
@@ -278,10 +295,13 @@ local function ignore_invalid(s)
   )
 end
 
-local function ignore_parse_error(s)
-  -- Ignore parse errors for unclosed codespan/optionlink/tag.
-  -- This is common in vimdocs and is treated as plaintext by :help.
-  return s:find("^[`'|*]")
+local function ignore_parse_error(s, fname)
+  local helpfile = vim.fs.basename(fname)
+  return (helpfile == 'pi_netrw.txt'
+    -- Ignore parse errors for unclosed tag.
+    -- This is common in vimdocs and is treated as plaintext by :help.
+    or s:find("^[*]")
```

Comment:
ignore_parse_error excludes fewer errors now, that's why help_spec tolerance increased. But the new parser actually has fewer total errors.